tools/doc/package.json

```diff
@@ -1,27 +1,6 @@
 {
-  "author": "Isaac Z. Schlueter <i@izs.me> (http://blog.izs.me/)",
-  "name": "node-doc-generator",
-  "description": "Internal tool for generating Node.js API docs",
-  "version": "0.0.0",
-  "engines": {
-    "node": ">=14.8.0"
-  },
-  "devDependencies": {
-    "highlight.js": "^11.11.1",
-    "js-yaml": "^4.1.0",
-    "rehype-raw": "^7.0.0",
-    "rehype-stringify": "^10.0.1",
-    "remark-frontmatter": "^5.0.0",
-    "remark-gfm": "^4.0.1",
-    "remark-html": "^16.0.1",
-    "remark-parse": "^11.0.0",
-    "remark-rehype": "^11.1.1",
-    "to-vfile": "^8.0.0",
-    "unified": "^11.0.5",
-    "unist-util-select": "^5.1.0",
-    "unist-util-visit": "^5.0.0"
-  },
-  "bin": {
-    "node-doc-generator": "generate.js"
+  "name": "doc",
+  "dependencies": {
+    "@node-core/api-docs-tooling": "github:nodejs/api-docs-tooling#734e5efeef14f16af45c12d30d89b8d3fa3afd34"
```

Comment:
Seems like a weird choice of ref, GH says that
https://github.com/nodejs/api-docs-tooling/commit/734e5efeef14f16af45c12d30d89b8d3fa3afd34 doesn't belong to the repo

Reply:
I think this was a temporary hash used for debugging. @flakey5 can you update?

Reply:
(FWIW The most recent `main` ref is 8c86289f155042d350f4dbb8d33c65824205737a)